docs/DeveloperGuide.adoc

```diff
@@ -895,35 +895,122 @@ what specific customers are ordering
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is FoodZoom and the *Actor* is the `user`, unless specified otherwise)
```

Comment:
Shouldn't the Actor be `Manager` and `Delivery man` ?

Reply:
makes sense... i was thinking more of the manager/deliveryman as a user

Reply:
hmm. I think since we used `Manager` and `Delivery man` inside our use cases, should be consistent.

Reply:
ok, will change.